devIocStats/devIocStatsString.c

```diff
@@ -274,8 +281,6 @@ static long epics_init_record(stringinRecord* pr)
           ppParam++;
         }
         pr->dpvt = 0;
-        recGblRecordError(S_db_badField,(void*)pr,
-                "devStringinEnvVar (init_record) Illegal INP parm field");
```

Comment:
Why is this being removed? It seems a bit inconsistent now with the other stringin init functions, each of which does print a similar error.